cEP-0010.md

```diff
@@ -136,10 +138,37 @@ Here is the list of aspects to be implemented along with their corresponding
 
 - `Root.Formatting`: This aspect checks on the format (the structure) of your
   codebase.
-- `Root.Formatting.Quotation`: This aspect checks on the quotations being used
-  in your files given `Root.Formatting.Quotation.preferred_quotation`.
+- `Root.Formatting.Quotation`: This aspect represents the quotation marks
+  being used in the files written in languages with multiple string delimiters,
+  which is common in recent languages, and excludes C-ish languages.
 - `Root.Formatting.Quotation.preferred_quotation`: This taste represents the
-  user's preferred quotation.
+  preferred quotation marks for the selected files. It ensures that every string
```

Comment:
this needs to clarify that the language definition will be used for each file.

Also, we need a section at the top of this cEP which explains that aspects need to rely on language definitions in order to avoid asking the user to supply tastes when the language restricts the available tastes.

Reply:
I thought that `preferred_quotation` takes `"` or `'` as argument. Are we going to use language definition to look at the default quotes that the language prefers? Aren't we going to use the quotes that the user asks for? I had designed this taste to map the `preferred_quotes` bear setting.

Please clarify this a bit. I think either you or me are little off track in this one.

Reply:
It is critical that we use language definition as a basis for aspects.

If a language only supports `"` as string literal delimiter, it is unacceptable for the aspect to allow it to be configured to use `'`.  The users preference is illogical in that case, and should be ignored.

e.g. if I have a global setting of `preferred_quotation = '`, it should only be operational in sub-sections which have a language that has multiple quotation (string delimiter) characters and only where `'` is one of the valid string delimiters.  We do not want to have to set the aspect settings in every section , otherwise they are no different from bear settings.

Reply:
I think now I am getting it. `preferred_quotation` takes `"` or `'` as argument but the setting applies only where the language supports it.

e.g. in C, even if the `preferred_quotation = '`, the setting won't be applied and the quotes would remain `"` only. So it more depends on the language preference.